configs/dgcnn/dgcnn_4xb32-cosine-100e_s3dis-seg-area4.py

```diff
@@ -0,0 +1,36 @@
+_base_ = [
+    '../_base_/datasets/s3dis-seg.py', '../_base_/models/dgcnn.py',
+    '../_base_/schedules/seg-cosine-100e.py', '../_base_/default_runtime.py'
+]
+
+# data settings
+train_area = [1, 2, 3, 5, 6]
```

Comment:
Two repeat codes in these configs with different splits. Suggest configs of test areas 1,2,3,4,6 directly  inherit from  config `dgcnn_4xb32-cosine-100e_s3dis-seg_test-area5.py`